package-parser/tests/processing/api/test_get_code_for_api_element.py

```diff
@@ -0,0 +1,67 @@
+import astroid
+import pytest
+
+from package_parser.processing.api._ast_visitor import trim_code
+
+
+@pytest.mark.parametrize(
+    "code_to_pare,expected_code",
+    [
+        (
+            """
```

Comment:
Might be worth adding a function like [`trim_margin`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/trim-margin.html) to make it clear which whitespace is relevant for the test, and which isn't.